src/components/AlertsTable.tsx

```diff
@@ -14,56 +14,68 @@ import {
 import { Tooltip, TooltipTrigger } from "@stacklok/ui-kit";
-import { getMaliciousPackage } from "@/lib/utils";
-import { Search } from "lucide-react";
-import { Markdown } from "./Markdown";
+import {
+  sanitizeQuestionPrompt,
+  parsingPromptText,
+  getIssueDetectedType,
+} from "@/lib/utils";
+import { KeyRoundIcon, PackageX, Search } from "lucide-react";
 import { useAlertSearch } from "@/hooks/useAlertSearch";
 import { useCallback } from "react";
 import { useNavigate, useSearchParams } from "react-router-dom";
 import { useFilteredAlerts } from "@/hooks/useAlertsData";
 import { useClientSidePagination } from "@/hooks/useClientSidePagination";
 
-const wrapObjectOutput = (input: AlertConversation["trigger_string"]) => {
-  const data = getMaliciousPackage(input);
-  if (data === null) return "N/A";
-  if (typeof data === "string") {
-    return (
-      <div className="bg-gray-25 rounded-lg overflow-auto p-4">
-        <Markdown>{data}</Markdown>
-      </div>
-    );
+const getTitle = (alert: AlertConversation) => {
+  const prompt = alert.conversation;
+  const title = parsingPromptText(
+    sanitizeQuestionPrompt({
+      question: prompt.question_answers?.[0]?.question.message ?? "",
+      answer: prompt.question_answers?.[0]?.answer?.message ?? "",
+    }),
+    prompt.conversation_timestamp,
+  );
+
+  return title;
+};
+
+function TypeCellContent({ alert }: { alert: AlertConversation }) {
+  const conversationType = alert.conversation.type;
+
+  switch (conversationType) {
+    case QuestionType.CHAT:
+      return "Chat";
+    case QuestionType.FIM:
+      return "Code Suggestion";
+    default:
+      return "Unknown";
   }
-  if (!data.type || !data.name) return "N/A";
+}
 
-  return (
-    <div className="max-h-40 w-fit overflow-y-auto whitespace-pre-wrap p-2">
-      <label className="font-medium">Package:</label>
-      &nbsp;
-      <a
-        href={`https://www.insight.stacklok.com/report/${data.type}/${data.name}`}
-        target="_blank"
-        rel="noopener noreferrer"
-        className="text-brand-500 hover:underline"
-      >
-        {data.type}/{data.name}
-      </a>
-      {data.status && (
+function IssueDetectedCellContent({ alert }: { alert: AlertConversation }) {
+  const issueDetected = getIssueDetectedType(alert);
+
+  switch (issueDetected) {
+    case "leaked_secret":
+      return (
         <>
-          <br />
-          <label className="font-medium">Status:</label> {data.status}
+          <KeyRoundIcon className="size-4" />
+          Blocked secret exposure
         </>
-      )}
-      {data.description && (
+      );
+    case "malicious_package":
+      return (
         <>
-          <br />
-          <label className="font-medium">Description:</label> {data.description}
+          <PackageX className="size-4" />
+          Blocked malicious package
         </>
-      )}
-    </div>
-  );
-};
+      );
+    default:
+      return "";
+  }
+}
```

Comment:
as far as I understand this should be a filter 🤔 for the charts, considering that it is not a filter right now, I wold suggest to avoid show this cell as link for now, considering also that there is a click login on the row, redireting to the chat

<img width="305" alt="Screenshot 2025-01-27 at 09 49 06" src="https://github.com/user-attachments/assets/0c1de035-d29a-453f-bd4d-aacc5ad990eb" />

Reply:
you mean clicking the link should activate a filter?

Reply:
as far as I understand from james yep
https://stacklok.slack.com/archives/C088Y05A85Q/p1737647833035189?thread_ts=1737644672.308539&cid=C088Y05A85Q

Reply:
![Screenshot_select-area_20250127110756](https://github.com/user-attachments/assets/e06ef629-b0b1-40df-99ad-59da3f7dd6bf)


and basically the solution is to make it look like this?

Reply:
I suppose this filtering is sort of doable with small changes already... albeit I think it would require changes to the filtering UI, so if possible in this PR I would "solve" it by not making those look like links

Reply:
agree